core/src/message/context.rs

```diff
@@ -814,18 +814,13 @@ mod tests {
             Error::OutgoingMessagesBytesLimitExceeded,
         );
     }
-
-    #[test]
+    /*     #[test]
```

Comment:
why?

Reply:
Since we got rid of payload storage in Dispatch MessageContext can't be created with payload bytes exceeding limit. This test is removed now